### ISSUE TITLE
[wasm][debugger] Debugger tests getting randomly aborted on CI

### ISSUE THREAD

**Issue by radical** [dotnet/runtime#89409]
[log](https://helixre107v0xdcypoyl9e7f.blob.core.windows.net/dotnet-runtime-refs-pull-89395-merge-93f9180bd3f44ef8bb/chrome-DebuggerTests.EvaluateOnCallFrameTests/1/console.2085d9ea.log?helixlogtype=result):

```
...
Detected a container, disabling sandboxing for debugger tests.
The active test run was aborted. Reason: Test host process crashed
Results File: /root/helix/work/workitem/uploads/xharness-output/logs/testResults.trx

Passed!  - Failed:     0, Passed:    85, Skipped:     0, Total:    85, Duration: 9 m 18 s - DebuggerTestSuite.dll (net8.0)
Test Run Aborted.
```

**Comment by lewing** [comment]
I bet we are taking longer than 100ms to close https://github.com/microsoft/vstest/blob/main/src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyOperationManager.cs#L325-L327

See the discussion at https://github.com/microsoft/vstest/issues/2952

**Comment by radical** [comment]
I haven't seen this issue since Jul 26. It would still be a good idea to change the timeout like @lewing suggested.

**Comment by radical** [comment]
This hasn't been hit on `main` again, but it is being seen on 7.0 .

**Comment by SamMonoRT** [comment]
hit again in https://github.com/dotnet/runtime/pull/91393

**Comment by SamMonoRT** [comment]
also in https://github.com/dotnet/runtime/pull/91664

**Comment by pavelsavara** [comment]
Hit again in #94299 [log](https://helixre107v0xdcypoyl9e7f.blob.core.windows.net/dotnet-runtime-refs-pull-94299-merge-d71f9208eed34257b7/chrome-DebuggerTests.EvaluateOnCallFrameTests/1/console.08664a2b.log?helixlogtype=result)

**Comment by pavelsavara** [comment]
> This hasn't been hit on `main` again, but it is being seen on 7.0 .

I see it on main [Log](https://helixre107v0xdcypoyl9e7f.blob.core.windows.net/dotnet-runtime-refs-pull-94299-merge-34f57efe4a224c04b7/chrome-DebuggerTests.EvaluateOnCallFrameTests/1/console.8111fca8.log?helixlogtype=result)

**Comment by radical** [comment]
Yeah, I have been seeing this on rolling builds pretty regularly :/

**Comment by pavelsavara** [comment]
See also https://github.com/dotnet/dnceng/issues/451

**Comment by jtschuster** [comment]
Renaming to track disabled test. The ErrorMessage is catching unrelated failures.
